composer.json

```diff
@@ -202,7 +202,7 @@
       "merge-extra-deep": false,
       "merge-scripts": true
     },
-    "wp-version": "6.0.3"
+    "wp-version": "6.1"
```

Comment:
We could upgrade straight to `6.1.1`, since it's a [maintainance release](https://wordpress.org/news/2022/11/wordpress-6-1-1-maintenance-release/).